.github/workflows/reviewd-ft.yml

```diff
@@ -0,0 +1,14 @@
+name: Functional Tests
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  functional-latest:
+    if: github.event.review.state == 'approved'
+    uses: vmware/repository-service-tuf/.github/workflows/functional.yml@main
+    with:
+      worker_version: dev
+      api_version: dev
+      cli_version: dev
```

Comment:
Yes, that is correct.
We want to identify if there is any incompatibility with the other components (development/"main" branch).
We, as maintainers, need to pay attention if the failure is something that breaks compatibility. The failure is not a blocker, but we need to identify it. It will help us:
- Discover early incompatibilities
- Identify correct order of merging components without breaking